R/ResampleDesc.R

```diff
@@ -141,6 +149,53 @@ makeResampleDescRepCV = function(reps = 10L, folds = 10L) {
   makeResampleDescInternal("repeated cross-validation", iters = folds * reps, folds = folds, reps = reps)
 }
 
+makeResampleDescFixedCV = function(horizon = 1L, initial.window = .5, size = 100L, skip = .01  ) {
+  horizon = asInteger(horizon, lower = 1L, upper = Inf)
+  assertNumeric(initial.window, lower = 1E-15, upper = 1)
```

Comment:
The true lower bound is data dependent anyways; if you add some arbitrary lower bounds check it will only confuse people or look like a mistake. Fun fact: every use of `.Machine$double.eps` in mlr is a bug.